### PR TITLE
feat: add vote deeplink to delegate pages

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Closes https://app.clickup.com/t/2qak9v1
Depends on releasing https://github.com/ArdentHQ/arkvault-url-php
Also, since requires compact tables in md screens it depends on https://github.com/ArdentHQ/arkscan/pull/68

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by alfonsobries** [comment]
@ItsANameToo new dependency https://github.com/ArkEcosystem/laravel-foundation/pull/457

**Comment by alfonsobries** [comment]
@ItsANameToo ✅

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
- at 360px the delegate page pushes the vote button away. Can you make it take some additional space from the delegate name instead? 
image

File: app/ViewModels/WalletViewModel.php [hunk 1]
```diff
@@ -69,4 +70,9 @@ public function nonce(): int
     {
         return $this->wallet->nonce->toNumber();
     }
+
+    public function voteUrl(): string
+    {
+        return ArkVaultUrlBuilder::get()->generateVote($this->address());
```
[the remark this answers is not in the document]
Comment (alfonsobries) [reply app/ViewModels/WalletViewModel.php]: @ItsANameToo :hide: fixed

**Comment by alfonsobries** [comment]
@ItsANameToo ✅

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**